src/dom/DomUtil.js

```diff
@@ -210,28 +212,37 @@ export function testProp(props) {
 // Resets the 3D CSS transform of `el` so it is translated by `offset` pixels
 // and optionally scaled by `scale`. Does not have an effect if the
 // browser doesn't support 3D CSS transforms.
-export function setTransform(el, offset, scale) {
+export function setTransform(el, offset, scale, bearing, pivot) {

```

Comment:
Too many arguments, here and in `setPosition`.
May be it'd better to handle all rotation-related stuff separately.
Like, store them as properties, and apply automatically.

Reply:
It is based on the [current version](https://github.com/Leaflet/Leaflet/blob/ce266a8c77731f83e8e9e674cf20712012f9ab19/src/dom/DomUtil.js#L187)

Reply:
I understand that you've only updated existing `rotate` branch.
Still I do not like that part.

Reply:
What do you mean by "handle all rotation-related stuff separately". Could you please expand on that? I don't quite understand what you mean by that.

Reply:
Just for example separate function `setRotation`.

Reply:
I got it :+1:

Reply:
I fixed it :)

Reply:
Well, I'm not sure that current implementation is convenient enough.
I do not like things like `DomUtil.setPositionAndRotation`.

I'd consider another approach: what if `setRotation` will just put `_leaflet_rotate` property into html el?
And `DomUtil.setPosition` will use that property to keep that rotation.

**This is just an idea, I haven't reviewed all the things as a whole!**

Reply:
I don't like that approach, I prefer to pass bearing as parameter